src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -193,6 +205,16 @@ private CommandResult executeCommand(String commandText) throws CommandException
                 handleExit();
             }
 
+            if (commandResult.isDisplayableCommandResult()) {
+                DisplayableCommandResult displayableCommandResult = (DisplayableCommandResult) commandResult;
```

Comment:
Actually, can we have this method in the abstract class and utilize polymorphism more? But this is fine as well haha